src/components/toast/Toast.tsx

```diff
@@ -0,0 +1,49 @@
+import { Snackbar, Alert } from "@mui/material";
+
+export enum ToastSeverity {
+  ERROR = "error",
+  INFO = "info",
+  SUCCESS = "success",
+  WARNING = "warning",
+}
+
+export enum ToastVariant {
+  FILLED = "filled",
+  OUTLINED = "outlined",
+  STANDARD = "standard",
+}
+
+type ToastProps = {
+  isOpen: boolean;
+  closeHandler: () => void;
+  severity: ToastSeverity | undefined;
```

Comment:
Why not optional param?

Reply:
Or if this can't be undefined then we can keep it like this
```suggestion
  severity: ToastSeverity;
```